pages/pool/[root]/[slug]/index.tsx

```diff
@@ -22,6 +24,9 @@ class Pool extends React.Component<Props> {
 
     return (
       <WithFooter>
+        <Head>
+          <title>Pool Overview: {pool.name} | Tinlake | Centrifuge | Decentralized Asset Financing</title>
```

Comment:
Maybe we can shorten the titles for the pools a bit? I think best practices for SEO titles are ~65 characters max. E.g. change it to `Pool Overview: {pool.name} | Tinlake | Centrifuge`

Reply:
Good point, done 👍